### DISCUSSION TITLE
[mpvnet] 如何设置1倍窗口大小打开视频？

### DISCUSSION THREAD

**Discussion by Ayane0709** [hooke007/mpv_PlayKit#143] [libmpv]
现在总是打开视频然后将窗口缩放设置成100%，那么怎么设置以100%大小打开视频呢？

**Comment by hooke007** [comment]
![Snipaste_2022-06-14_17-12-36](https://user-images.githubusercontent.com/41094733/173625475-52431caa-6a1a-4392-a8b7-4a7f9b8a1b21.jpg)

**Reply by Ayane0709** [reply]
很奇怪，这样设置了之后，打开的视频画面大小总是比源视频小一点，比如1920X1080的视频打开后只有18XX乘以1020左右大小，还是要在画面比例里设置一下1倍率才正常。

**Reply by hooke007** [reply]
你的桌面分辨率多少，你的autofit-larger多少

**Reply by Ayane0709** [reply]
已解决，感谢，autofit-larger设置成100就好了
